examples/application/main.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"../../iis"
+)
+
+func main() {
+	log.Printf("Starting test...")
+
+	testAppPath := "C:\\inetpub\\testing"
+	testAppContent := "<html><head><title>IIS Testing</title></head><body><h1>Hello World!</h1></body></html>"
+	appPoolName := "testing"
+	applicationName := "testing"
+	websiteName := "Default Web Site"
```

Comment:
so that this example runs cleanly every time (and has no dependencies) - could we make these dynamic:

```
rInt := helpers.RandomInt()
appPoolName := fmt.Sprintf("app-pool-%d", rInt)
websiteName := fmt.Sprintf("website-%d", rInt)
applicationName := fmt.Sprintf("application-%d", rInt)
```